wandb: Using wandb-core as the SDK backend. Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: takonoselidze (takonoselidze-charles-university). Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.18.3
wandb: Run data is saved locally in /scratch.ssd/takonoselidze/job_5312938.pbs-m1.metacentrum.cz/wandb/run-20241016_123606-53ufb5up
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run logical-feather-1
wandb: ‚≠êÔ∏è View project at https://wandb.ai/takonoselidze-charles-university/Cleverhans%20FGSM-resnet50
wandb: üöÄ View run at https://wandb.ai/takonoselidze-charles-university/Cleverhans%20FGSM-resnet50/runs/53ufb5up
Traceback (most recent call last):
  File "main.py", line 274, in <module>
    wandb.log(f'\nAverage success rate of the attack on {mod_name} model: {Nmis/num_of_images* 100:.2f}%\n')
  File "/storage/praha1/home/takonoselidze/.local/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 452, in wrapper
    return func(self, *args, **kwargs)
  File "/storage/praha1/home/takonoselidze/.local/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 403, in wrapper_fn
    return func(self, *args, **kwargs)
  File "/storage/praha1/home/takonoselidze/.local/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 393, in wrapper
    return func(self, *args, **kwargs)
  File "/storage/praha1/home/takonoselidze/.local/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1934, in log
    self._log(data=data, step=step, commit=commit)
  File "/storage/praha1/home/takonoselidze/.local/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1648, in _log
    raise ValueError("wandb.log must be passed a dictionary")
ValueError: wandb.log must be passed a dictionary
